palo_alto_cortex_xdr/help.md

```diff
@@ -0,0 +1,334 @@
+# Description
+
+Stop modern attacks with the industry’s first extended detection and response platform that spans your endpoint, network and cloud data.
+
+# Key Features
```

Comment:
I think we need to update it since we added new actions.

Reply:
Fixed